gene_finder.py

```diff
@@ -9,6 +9,7 @@
 import random
 from amino_acids import aa, codons, aa_table   # you may find these useful
 from load import load_seq
+dna = load_seq("./data/X73525.fa")
```

Comment:
While the logic here is similar for both `import` and `load_seq`, the former is a module import statement while the latter is a function call. It is conventional to keep `import` statements separate from the rest of the code. In this case, `load_seq` should be in `__main__`